ProjectLighthouse.Servers.Website/Extensions/FormattingExtensions.cs

```diff
@@ -5,7 +5,41 @@ namespace LBPUnion.ProjectLighthouse.Servers.Website.Extensions;
 
 public static class FormattingExtensions
 {
-    public static string GetLevelLockIcon(this SlotEntity slot) => slot.InitiallyLocked ? "icon lock" : "";
+    private static string GetLevelLockIcon(this SlotEntity slot) => slot.InitiallyLocked ? "ui black icon lock" : "";
+    private static string GetTeamPickedIcon(this SlotEntity slot) => slot.TeamPick ? "ui pink icon certificate" : "";
+
+    // ReSharper disable once UnusedParameter.Global
+    private static string GetLevelWarningIcon
+        (this SlotEntity slot) =>
+        slot.Lbp1Only || slot.CrossControllerRequired || slot.MoveRequired ? "ui orange icon exclamation circle" : "";
```

Comment:
Since you are relying on html title tags, this implementation doesn't translate well to mobile. You may need to create more descriptive icons.